### PR TITLE
Basic newfeed and create basic post

### PR DESCRIPTION
**Pull request by manhdat22**
New feed and post basic Post

### CODE REVIEW

**Comment by manhdat22** [comment]
ready

**Comment by manhdat22** [comment]
ready

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class PostsController < ApplicationController
+  def create
+    @post = current_user.post.build(post_params)
```
Comment (thangtx88) [inline app/controllers/posts_controller.rb]: bỏ ()

**Comment by manhdat22** [comment]
ready

File: app/views/shared/_feed.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <br>
+      <p>
+        <b>
+          <%= image_tag "resources/avatar-placeholder.png",width:  50, height: 50 %>
+          &nbsp;<%= link_to p.user.username, user_path(p.user_id) %>
+        </b>
+      </p>
+      <%= image_tag "upload/post/sample.jpg",width:  490 %>
+      <br><br>
+      <a href="#">Like</a>
```
Comment (thangtx88) [inline app/views/shared/_feed.html.erb]: I18n

File: app/views/shared/_feed.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      <a href="#">Comment</a>
```
Comment (thangtx88) [inline app/views/shared/_feed.html.erb]: I18n

**Comment by thangtx88** [comment]
Commented

**Pull request closed without merging**